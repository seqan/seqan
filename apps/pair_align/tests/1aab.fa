>hmgl_trybr
KKDSNAPKRAMTSFMFFSSDFRSKHSDLSIVEMSKAAGAAWKELGPEERKVYEEMAEKDKERYKREM
>hmgt_mouse
KPKRPRSAYNIYVSESFQEAKDDSAQGKLKLVNEAWKNLSPEEKQAYIQLAKDDRIRYDNEMKSWEEQMA
E
