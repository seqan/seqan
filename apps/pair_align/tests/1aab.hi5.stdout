Alignment score: -2147483648
